src/python2js.c

```diff
@@ -204,10 +207,43 @@ _python2js(PyObject* x)
   }
 }
 
+int
+_python2js_cache(PyObject* x, PyObject* map, PyObject* pyparent, int jsparent)
```

Comment:
A small comment with the high level goal of this function would help readability.